Common/archivereader.h

```diff
@@ -109,22 +89,22 @@ class FileReader
 	void reset();
 
 	/**
-	 * @brief isClosed Checks for file/archive not open.
-	 * @return true if file closed.
+	 * @brief isOpen Checks for file/archive being open.
+	 * @return true if file opened...
 	 */
-	bool isClosed();
+	bool isOpen();
```

Comment:
This guy was left empty. I think you are really used to Qt taking care of these! 🙃

Reply:
Oh haha, I guess I messed up the "move to archivereader.h" together with renaming the file or something :p

Merci